change/@ni-nimble-components-49250ef8-26bc-404b-bba7-342b1f3b6a8e.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Looks good, just make sure to not accept the unstable menu story change. Captured some notes on getting passed unstable stories: https://github.com/ni/nimble/issues/1479#issuecomment-2341882851